package.json

```diff
@@ -41,20 +42,14 @@
     "singleQuote": true,
     "semi": false
   },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
+  "browserslist": [
+    ">0.2%",
+    "not dead",
+    "not op_mini all"
+  ],
```

Comment:
This change was necessary to support new dependencies which made use of `??`, otherwise babel would throw an unexpected token error.